### DISCUSSION TITLE
Problem with incoming email

### DISCUSSION THREAD

**Discussion by warstrolo** [postalserver/postal#3042] [Help with using Postal] [answered]
Hello, when I receive incoming email send to postman managed domain
The smtp service send a "530 Authentication required"

EDIT : I tried using the route method and it's working when I'm using, but is there a way to keep the message in postman ?

**Comment by warstrolo** [comment, marked as answer]
![image](https://github.com/postalserver/postal/assets/58334832/93dfaa89-0bd4-4338-ac61-9a26e15df1c1)
Never mind I have found the answer

(for those who are searching the answer is there)

Other question is it possible to have a catch all address ?
=> Yes
I created three rules in this order
test@lamedusegroup.fr => forward to my email
*@lamedusegroup.fr => message are held
blbl@lamedusegroup.fr => forward to my email

Partial catch address is not supported e.g. "blb*@lamedusegroup.fr"

I will leave this here for people to find the answer when searching the discussion thread

**Reply by willpower232** [reply]
catch-all addresses are dealt with like this

![image](https://github.com/user-attachments/assets/9057f1a6-92ad-46d0-bcbf-6310fc9b76c1)

It may be worth pointing out that Postal isn't designed to store messages long term so may or may not be suitable for what you're doing.
